CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Thanks for considering contributing and making our planet easier to explore!
+
+We're excited you would like to contribute to Cumulus! Whether you're finding bugs, adding new features, fixing anything broken, or improving documentation, get started by submitting an issue or pull request!
+
+## Submitting an Issue
+
+If you have any questions or ideas, or notice any problems or bugs, first [search open issues](https://github.com/cumulus-nasa/cumulus-message-adapter/issues) to see if the issue has already been submitted. We may already be working on the issue. If you think your issue is new, you're welcome to [create a new issue](https://github.com/cumulus-nasa/cumulus-message-adapter/issues/new).
+
+## Pull Requests
+
+If you want to submit your own contributions, follow these steps:
+
+* Fork the Cumulus Message Adapter repo
+* Create a new branch from the branch you'd like to contribute to
+* If an issue does't already exist, submit one (see above)
+* [Create a pull request](https://help.github.com/articles/creating-a-pull-request/) from your fork into the target branch of the cumulus-nasa/cumulus repo
```

Comment:
This should be `of the cumulus-nasa/cumulus-message-adapter` repo?

Reply:
Ahhhh I didn't change that one on any of them!! Good catch, thank you!